securedrop/journalist.py

```diff
@@ -216,12 +216,14 @@ def admin_new_user_two_factor():
         token = request.form['token']
         if user.verify_token(token):
             flash(
-                "Two-factor token successfully verified for user {}!".format(
+                "Token in two-factor authentication "
+                "accepted for user {}.".format(
                     user.username),
                 "notification")
             return redirect(url_for("admin_index"))
         else:
-            flash("Two-factor token failed to verify", "error")
+            flash("Could not verify token in two-factor authentication.",
```

Comment:
This used to be a comma. It could be a dot, but it would make it "Error" to follow.